src/components/FormHtmlEditor.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class='form-group'>
+    <div :class="classList">
+      <editor
+        v-model="content"
+        :init="editorSettings"
+        v-bind:disabled="disabled"
+        @onFocus="$emit('onFocus')"
+        @onBlur="$emit('onBlur')">
```

Comment:
This may be a good use case for using `$listeners`. See https://vuejs.org/v2/api/#vm-listeners and https://vuejs.org/v2/guide/components-custom-events.html#Binding-Native-Events-to-Components.